src/main/groovy/com/avioconsulting/mule/linter/model/MuleComponent.groovy

```diff
@@ -2,14 +2,21 @@ package com.avioconsulting.mule.linter.model
 
 class MuleComponent {
 
-    private String name
-    private Map<String, String> attributes = [:]
+    private final String componentName
+    private final String componentNamespace
+    private final Map<String, String> attributes = [:]
     private final List<MuleComponent> children
 
     MuleComponent(Map<String, String> attributes) {
```

Comment:
We shouldn't be able to create an object without a componentName and Namespace, they all have one and there is no default

Reply:
removed this construction